parking-lot-help-request/firmware/README.md

```diff
@@ -0,0 +1,99 @@
+2. The [Notecard Playground](https://dev.blues.io/notecard-playground/): an in-browser terminal that emulates much of the functionality of the Notecard CLI. 
+
+Both of these tools are accessible through the Blues [Developer Experience site](https://www.dev.blues.io).
+
+> **NOTE**: If you _do_ choose to use the in-browser terminal, you'll need to do so on a Chrome-based web browser like Chrome or Edge that supports the web serial API. 
+
+![Screenshot of the in-browser Notecard Playground on dev.blues.io](./readme-notecard-playground.png)
+_The in-browser Notecard Playground available on dev.blues.io._
+
+The instructions to configure the Notecard will assume the Notecard CLI is being used, but this will also work with a very slight modification if you choose to use the in-browser Notecard Playground instead.
+
+1. Before you can configure your Notecard, you must set up a free [Notehub.io](https://notehub.io) account and create a project where the Notecard will send its data to. Follow the [Set Up Notehub quickstart](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-f/#set-up-notehub) to sign up and create your first project if you haven't done so already. Be sure to copy the Project UID after creating it - you'll need it for the Notecard configuration script.
+2. Download and install the [Notecard CLI](https://dev.blues.io/tools-and-sdks/notecard-cli/#installation) from the Blues Developer Experience site.
+3. Plug the Notecarrier containing the Notecard into your computer with a USB-A to micro USB cable.
+4. Copy the JSON file example in this folder named [`parking-lot-help-request-config-script.json`](./parking-lot-help-request-config-script.json) to your local machine.
+5. Remove all the # comments describing what each command does, replace the `product` and `sn` placeholders in the script with your Notehub Project UID (copied earlier when you created the project) and preferred Notecard serial number (e.g. `Lot G` or `G14` - depending on how location specific you want your help request buttons to be), and resave the file. It will resemble the code snippet below when you're done.
+```json
+{"req":"env.default","name":"parking_lot_help_request_config_script_version","text":"1.1.1"}
+{"req":"hub.set","product":"com.your-company.your-name:your_product", "sn":"preferred_device_name_here"}
+{"req":"hub.set","mode":"minimum","outbound":1440,"inbound":10080}
+{"req":"card.aux","mode":"neo-monitor"}
+{"req":"card.voltage","mode":"lipo"}
+{"req":"card.motion.mode","stop":true}
+```
+6. Upload the configuration script to the Notecard via the Notecard CLI by running the following command from the command line:
+```bash 
+$ notecard -setup parking-lot-help-request-config-script.json`
+```
+7. This will execute each of the JSON commands line by line against the Notecard and then it should be all set as far as firmware configuration goes.
```

Comment:
Suggested wording tweak you can take or leave ⬇️ 

This executes each command on your connected Notecard, one at a time. When the setup completes, your Notecard’s configuration is complete.

Reply:
Nice